h2o-core/src/main/java/water/rapids/Env.java

```diff
@@ -211,7 +211,6 @@ static void init(AstPrimitive ast, String name) {
     init(new AstTable());
     init(new AstUnique());
     init(new AstVariance());
-    init(new AstSpearman());
```

Comment:
this smells little bit like breaking api change? could someone be affected by removing Spearman support from rapids?

Reply:
1. It has been there since `3.28.0.1`, where I implemented it only as Rapids. Which means if there is someone using Rapids directly, it could break their code without javac catching it (all plain strings).
2. As @michalkurka told me, users are not meant to write Rapids expressions on their own, those are assembled by our API. Which means to me this contract is not guaranteed, structure of Rapids is not guaranteed and it can be joined together with `cor` Rapids.

Reply:
@michalkurka WDYT ? I ask the very same question in the description of this PR. I'm for removing the old way myself.

Reply:
I think the core part here is `users are not meant to write Rapids expressions` if that is true than this is fine

Reply:
this should be fine